<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>577</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Narrow</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ion Identification Interface</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{color: &quot;#23373B&quot;}
QCheckBox{background-color: &quot;#FAFAFA&quot;; color: &quot;#23373B&quot;}
QTextEdit{color: &quot;#23373B&quot;}
QMainWindow{background-color: &quot;#FAFAFA&quot;}
QWidget{background-color: &quot;#FAFAFA&quot;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
   <widget class="GraphicsLayoutWidget" name="gSpectrum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>881</width>
      <height>350</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTableView" name="ionTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>881</width>
      <height>171</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTreeView" name="fileList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>150</y>
      <width>341</width>
      <height>394</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="1">
      <widget class="QCheckBox" name="HLikeCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>H-like</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="calibratePeakLocLab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Peak Loc. [kHz]</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="HeLikeCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>He-like</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="spanInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="isometricStateLab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Isometric state (e.g. 0, 1)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="ionInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QCheckBox" name="fileCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>File Load</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="calibratePeakLocInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="cenFreqInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="calibrateBrhoCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Brho [Tm]</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="ionButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="ionCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ion (e.g. 3H1,3H,H1,3,H,1+)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="isometricStateInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="LiLikeCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Li-like</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="spanLab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>span [kHz]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="calibrateBrhoInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="cenFreqLab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>center frequency [MHz]</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="calibrateButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Calibrate</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="bareCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bare</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="calbirateHarmLab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Harmonic</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="calibrateHarmInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
